state_machine/applications/system-check/tests/current_test.py

```diff
@@ -0,0 +1,49 @@
+from lib.pycubed import cubesat
+import time
+
+def current_test():
+    measurements = 100
+    vals = [0] * measurements
+    outlier_count = 0
+    for i in range(measurements):
+        vals[i] = cubesat.battery_current
+        time.sleep(0.05)
+    sum = 0
+    for val in vals:
+        sum += val
+    avg = sum / 100
+    for val in vals:
+        if not ((avg - 2) <= val <= (avg + 2)):
```

Comment:
What are the units on this? I assume mA? 

I'd recommend adding a variable `outlier_threshold = 2`.

Reply:
mA